tests/test_skip_pg_hba.py

```diff
@@ -0,0 +1,204 @@
+import pgautofailover_utils as pgautofailover
```

Comment:
test case naming could be test_cert_usage. Most of the file contents is related to certificate creation and usage.

Reply:
We actually test `--skip-pg-hba` here, which requires making sure a customer can come up with their own full implementation of the authentication needs for pg_auto_failover, from within their own provisioning system. I chose to implement SSL based authentication with the `cert` authentication method, because I think that's the most secure alternative and having an example how to do that might be good for the customers.